ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -117,16 +117,6 @@ object ParallelIndexerFactory {
             _ <- meteringAggregator(dbDispatcher)
           } yield dbDispatcher
         ) { dbDispatcher =>
-          val stringInterningView = new StringInterningView(
```

Comment:
unfortunately there was a reason why we instantiated the view for each indexing session: this view is being populated ahead (interning entries which are not persisted yet), which means we need to "roll it back", as we restart indexing. This was achieved with reinitialization. We probably need to make interning smarter to support this use case here (the same interning for both indexer and index)

Reply:
Why do we need to roll it back? Are we expecting different incoming data from the updates stream? As I see insertion into the StringInterningView is idempotent (https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/scala/platform/store/interning/RawStringInterning.scala#L31)